katib-ui-rock/rockcraft.yaml

```diff
@@ -19,36 +19,86 @@ platforms:
   amd64:
 
 parts:
-  katib-ui:
+  frontend-lib:
+    plugin: nil
+    source: https://github.com/kubeflow/kubeflow
+    source-type: git
+    # Set source-tag here to the commit cited in https://github.com/kubeflow/katib/blob/master/pkg/ui/v1beta1/frontend/COMMIT
+    # for this Katib release
+    # Note: Katib 0.16.0 cannot build with Kubeflow v1.8-branch because Kubeflow refactored this lib
+    source-commit: 046c6d3c8301a95a2809c0d927bbbf72638dd639
+    # Omit source-depth because we're cloning a commit
+    # source-depth: 1
```

Comment:
I'll delete them.  We cannot uncomment `source-depth` (because it is incompatible with `source-commit`, which checks out a single commit), but leaving this here at all really isn't needed.